news.xml

```diff
@@ -1,6 +1,65 @@
 <?xml version="1.0" encoding="utf-8" ?>
 
 <news>
+  <item>
+    <pubDate>TODO</pubDate>
+    <title>
+      NixOS 20.03 released
+    </title>
+    <description>
+      <a href="https://github.com/NixOS/nixos-artwork/blob/master/releases/20.03-markhor/markhor.png">
+        <img class="inline" src="logo/nixos-logo-20.03-markhor-lores.png" alt="20.03 Markhor logo" with="100" />
+      </a>
+      <p>
+        Hello everyone, I am <a href="https://discourse.nixos.org/u/worldofpeace">worldofpeace</a>
+        20.03 release manager. I am announcing our new stable release: NixOS
+        20.03 “Markhor”.
+      </p>
+      <ul>
+        <li><a href="https://nixos.org/nixos/manual/release-notes.html#sec-release-20.03">Release notes</a></li>
+        <li><a href="https://nixos.org/nixos/manual/index.html#sec-upgrading">Upgrade instructions</a></li>
+        <li><a href="https://nixos.org/download.html">Image downloads</a></li>
+      </ul>
+      <h3>Some Numbers</h3>
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Count</th>
+            <th>Author</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>TODO</td>
+            <td>TODO</td>
+          </tr>
+        </tbody>
+      </table>
```

Comment:
this section is replaced in my markdown at https://hackmd.io/Uy24aMgYThyPIHZWyXGRcA?both
"## NixOS 20.03 Contributors ### Top 30 ordered by commits [TABLE]`